client/ayon_core/hosts/unreal/api/hierarchy.py

```diff
@@ -0,0 +1,289 @@
+from pathlib import Path
+from qtpy import QtWidgets, QtCore, QtGui
+
+from ayon_api import get_folders_hierarchy
+from ayon_core import (
+    resources,
+    style
+)
+from ayon_core.pipeline import get_current_project_name
+from ayon_core.settings import get_project_settings
+from ayon_core.tools.utils import (
+    show_message_dialog,
+    PlaceholderLineEdit,
+    SquareButton,
+)
+from ayon_core.tools.ayon_utils.widgets import (
```

Comment:
NOTE: This has changed to `from ayon_core.tools.utils import SimpleFoldersWidget` from this PR https://github.com/ynput/ayon-core/pull/195